src/account/AccountPickScreen.js

```diff
@@ -1,7 +1,7 @@
 /* @flow strict-local */
 
 import React, { PureComponent } from 'react';
-
```

Comment:
nit: Keep a blank line, and keep third-party imports (the one from 'react' and the new one from 'react-native') above the blank line.